cli/client.go

```diff
@@ -768,10 +768,10 @@ func parseBaseURL(baseURL string, verifyConnection bool) (*url.URL, []rpc.DialOp
 
 	if verifyConnection {
 		// Check if URL is even valid with a TCP dial.
-		conn, err := net.DialTimeout("tcp", baseURLParsed.Host, 3*time.Second)
+		conn, err := net.DialTimeout("tcp", baseURLParsed.Host, 10*time.Second)
```

Comment:
sorry, late to the review @benjirewis - we might want to consider "elevating" this parameter to a constant in the package.

the immediate change in this PR is fine tho :+1:

Reply:
yeah that's fine, no need for a ticket here